lib/components/Editor/CustomExtensions/Variable/index.js

```diff
@@ -20,12 +21,11 @@ const Variables = ({ editor, variables }) => {
   return (
     <Dropdown
       customTarget={() => (
-        <button
-          type="button"
-          className="neeto-editor-fixed-menu__item"
-        >
-          <HashtagFilled size={MENU_ICON_SIZE} />
-        </button>
+        <ToolTip content="Variables" position="bottom">
```

Comment:
Since now we have started using Tooltip for multiple button and may be more in future, I think it is better to create a common Button component that can accept icon and tooltip props just like how we have done in neetoUI components. Please check the feasiblity for the same